NinjaResidence/GAMEMANAGER.h

```diff
@@ -20,12 +35,13 @@ static class GAMEMANAGER
 	static void ChangeDisplayMode(void);
 	static LRESULT CALLBACK WndProc(HWND hWnd, UINT msg, WPARAM wp, LPARAM lp);
 private:
+	const char ApiName[10] = "�E�҉��~";
 	static DirectX* pDirectX;
 	static SceneManager* pSceneManager;
 	static HWND hWnd;
 	static bool WinMode;	//true:Window�@false:Full
 	static bool DeviceLost;
 	static RECT WinRect;			//Window Mode �ł̈ʒu�傫��
-
+	static SoundLib::SoundsManager* pSoundManager;
```

Comment:
上で `using SoundLib::SoundsManager` と書くのなら
```suggestion
	static SoundsManager* pSoundManager;
```
で良いのでは？

Reply:
using宣言は後で加えたので
指摘箇所の”SoundLib::”は消し忘れていました。

Reply:
指摘箇所消去しました。

Reply:
消去しました。